src/Queries/QueryRunner/QueryRunner.ts

```diff
@@ -41,6 +41,36 @@ export type Neo4jSupportedProperties = Record<
   Neo4jSupportedTypes | undefined
 >;
 
+/** symbols for update operations */
+const OpRm: unique symbol = Symbol('remove');
+
+export const UpdateOp = {
+  remove: OpRm,
+} as const;
+
+export type UpdateTypes = {
```

Comment:
Do you think we should not export this? It's only used here, and we can skip exporting it on the package to avoid confusion. What do you think? 🤔

Reply:
It's used in `ModelOps.ts` to create the `UpdateDataI` type. Do you think we should move `UpdateDataI` to `QueryRunner.ts`?

Reply:
I see. Let's keep it here. I'll resolve to get this feature out, but in the future I might change some things to have a better control of what's exported to the package user. Right now I just export everything.